tests/integration/api_v3/http/generators.test.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+const { api } = require('../../../helpers/api');
+
+const {
+	badRequestSchema,
+	goodRequestSchema,
+	metaSchema,
+} = require('../../../schemas/httpGenerics.schema');
+
+const {
+	generatorSchema,
+} = require('../../../schemas/api_v3/generatorSchema.schema');
+
+const baseUrl = config.SERVICE_ENDPOINT;
+const endpoint = `${baseUrl}/api/v3`;
+
+describe('Generators API', () => {
+	describe('GET /generators', () => {
+		it('limit = 100 -> ok', async () => {
```

Comment:
Please add a few more test cases without any request params, with limit, offset and a combination of both.